.gitpod.yml

```
@@ -1,11 +1,13 @@
 tasks:
   - init: |
-      mvn -am -pl war,bom -P quick-build clean install
+      mvn -B -T 1C -DskipTests package
```

Please explain why you replaced the command from the contributing guide with a different command?
```suggestion
      mvn -B -T 1C -DskipTests package
```